kevm-pyk/src/kevm_pyk/kompile.py

```diff
@@ -121,9 +121,11 @@ def kevm_kompile(
     llvm_kompile_type: LLVMKompileType | None = None,
     enable_llvm_debug: bool = False,
     debug: bool = False,
+    md_selector: str | None = None,
 ) -> Path:
     backend = target.backend
-    md_selector = 'k & ! standalone' if target == KompileTarget.NODE else 'k & ! node'
+    if not md_selector:
+        md_selector = 'k & ! standalone' if target == KompileTarget.NODE else 'k & ! node'
```

Comment:
Hmmmm, are we sure we need to thread through this option to the user? I think the target actually fully determines what the `md_selector` needs to be, so we can just eliminate the option to `kevm_kompile`